### PR TITLE
Tweaks to the script to replace AppVeyor with the new GitHub builds.

### LINKED ISSUE #4: the script doesn't seem to work anymore, it looks like the build updates system has changed. how can you fix it?
image

### PR DESCRIPTION
**Pull request by essasetic**
As the name implies, it makes slight tweaks to the script in-order to accommodate the sudden switch for Ryujinx using github.

This is based on the testing branch so if you want me to merge it there or base it on the master branch or if you're perfectly fine with merging it anyway let me know.

The script should work as it did before.

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +8,7 @@ The purpose of the installer is to place Ryujinx inside ~/.local/share/Ryujinx,
 
 Usage:
 Open a terminal, paste <br>
-`bash -c "$(curl -s https://raw.githubusercontent.com/edisionnano/Pine-jinx/main/pinejinx.sh)"` <br>
+`bash -c "$(curl -s https://raw.githubusercontent.com/essasetic/Pine-jinx/main/pinejinx.sh)"` <br>
```
Comment (MGThePro) [inline README.md]: Might want to change this back to the original repo for the PR

File: pinejinx.sh [hunk 2]
```diff
@@ -98,41 +113,46 @@ install () {
...
 	printf "Also don't forget to show your love on Patreon at https://www.patreon.com/ryujinx\n"
 }
-uninstall () {
+uninstall() {
 	printf "Uninstalling..."
 	rm -rf /home/${USER}/.local/share/Ryujinx
 	rm -rf /home/${USER}/.local/share/mime/packages/Ryujinx.xml
 	rm -rf /home/${USER}/.local/share/applications/Ryujinx.desktop
 	rm -rf /home/${USER}/.local/share/icons/Ryujinx.png
+	rm -rf /home/${USER}/.config/gamemode.ini
```
Comment (MGThePro) [inline pinejinx.sh:125]: I know this change is from the testing branch, but I don't think it's safe to assume that deleting the gamemode config is fine on every system. Same goes for overwriting it in line 75. Ideally there would be a way to pick a custom config just for ryujinx via env variable or parameter, but I'm not sure if that's an option

File: README.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx!**
-
+**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx! Also please note that this fork is a heavy WIP and only grabs 1.1.4 as of now. This only affects the master build, not the LDN build (as its hosted on patreon)**
```
Comment (MGThePro) [inline README.md]: This also seems outdated

File: README.md (hunk 1 above)
Comment (essasetic) [reply README.md]: Ah yeah, oversight on my end.

File: README.md (hunk 3 above)
Comment (essasetic) [reply README.md]: Yeah I'll change that back. I only made it when I began work on it last night.

File: pinejinx.sh (hunk 2 above)
Comment (essasetic) [reply pinejinx.sh:125]: Yeah I agree, but I'm simply hotfixing the script to make it work with github and don't really have any experience with shell. So I'll have to bring that one up to the maintainer.

Comment (MGThePro) [reply pinejinx.sh:125]: removing line 125 and 75 should be sufficient, I don't see the script touching the gamemode.ini anywhere else

**Comment by edisionnano** [comment]
I'm not yet ready to push testing to main. First of all because adding chmod on the desktop file breaks GNOME.

**Comment by essasetic** [comment]
> I'm not yet ready to push testing to main. First of all because adding chmod on the desktop file breaks GNOME.

Removed that from the desktop file, also added everything else from the testing branch as I somehow forgot to do so beforehand

**Comment by essasetic** [comment]
image

Don't merge yet. This is a big issue.

**Comment by essasetic** [comment]
> image
> 
> Don't merge yet. This is a big issue.

Ok so it seems manually removing the /tmp/pineapple file solves this but I'm not sure why the program isn't doing this automatically. It could be solved now maybe?

EDIT: Applied the hardcoded 1.1.4 version, installed it, then undid the changes and got 1.1.5 installed with no tar errors. Think we're good to go!

**Pull request merged**